bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -385,6 +386,25 @@ static int get_string_or_msg_field_size(struct op_context *ctx, const ProtobufCF
     return MAP_VAL_STR_SIZE;
 }
 
+static int get_msg_field_size(const ProtobufCFieldDescriptor *field)
+{
+    return ((ProtobufCMessageDescriptor *)(field->descriptor))->sizeof_message;
+}
+
+static int get_field_size(struct op_context *ctx, const ProtobufCFieldDescriptor *field)
+{
+    char *value = NULL;
+
+    if (field->type == PROTOBUF_C_TYPE_MESSAGE)
+        return get_msg_field_size(field);
+
+    if (field->type == PROTOBUF_C_TYPE_STRING) {
```

Comment:
The case `field->type == PROTOBUF_C_TYPE_STRING` is already excluded in line 519, so do we still need to do this case on this side?